tests/core/mempool/test_mempool_manager.py

```diff
@@ -329,3 +331,61 @@ async def test_sb_twice_with_eligible_coin_and_different_spends_order() -> None:
+        (ConditionOpcode.ASSERT_SECONDS_RELATIVE, 0, MempoolInclusionStatus.SUCCESS, None),
+        (ConditionOpcode.ASSERT_SECONDS_RELATIVE, 1, MempoolInclusionStatus.FAILED, Err.ASSERT_SECONDS_RELATIVE_FAILED),
+        (ConditionOpcode.ASSERT_HEIGHT_RELATIVE, -2, MempoolInclusionStatus.SUCCESS, None),
+        (ConditionOpcode.ASSERT_HEIGHT_RELATIVE, -1, MempoolInclusionStatus.SUCCESS, None),
+        (ConditionOpcode.ASSERT_HEIGHT_RELATIVE, 0, MempoolInclusionStatus.PENDING, Err.ASSERT_HEIGHT_RELATIVE_FAILED),
+        (ConditionOpcode.ASSERT_HEIGHT_RELATIVE, 1, MempoolInclusionStatus.PENDING, Err.ASSERT_HEIGHT_RELATIVE_FAILED),
+        (ConditionOpcode.ASSERT_HEIGHT_ABSOLUTE, 4, MempoolInclusionStatus.SUCCESS, None),
+        (ConditionOpcode.ASSERT_HEIGHT_ABSOLUTE, 5, MempoolInclusionStatus.SUCCESS, None),
+        (ConditionOpcode.ASSERT_HEIGHT_ABSOLUTE, 6, MempoolInclusionStatus.PENDING, Err.ASSERT_HEIGHT_ABSOLUTE_FAILED),
+        (ConditionOpcode.ASSERT_HEIGHT_ABSOLUTE, 7, MempoolInclusionStatus.PENDING, Err.ASSERT_HEIGHT_ABSOLUTE_FAILED),
+        (ConditionOpcode.ASSERT_SECONDS_ABSOLUTE, TEST_TIMESTAMP - 1, MempoolInclusionStatus.SUCCESS, None),
+        (ConditionOpcode.ASSERT_SECONDS_ABSOLUTE, TEST_TIMESTAMP, MempoolInclusionStatus.SUCCESS, None),
+        (
+            ConditionOpcode.ASSERT_SECONDS_ABSOLUTE,
+            TEST_TIMESTAMP + 1,
+            MempoolInclusionStatus.FAILED,
+            Err.ASSERT_SECONDS_ABSOLUTE_FAILED,
+        ),
+        (
+            ConditionOpcode.ASSERT_SECONDS_ABSOLUTE,
+            TEST_TIMESTAMP + 2,
+            MempoolInclusionStatus.FAILED,
+            Err.ASSERT_SECONDS_ABSOLUTE_FAILED,
+        ),
+    ],
+)
+async def test_ephemeral_timelock(
+    opcode: ConditionOpcode,
+    lock_value: int,
+    expected_status: MempoolInclusionStatus,
+    expected_error: Optional[Err],
+) -> None:
+    mempool_manager = await instantiate_mempool_manager(
+        get_coin_record=get_coin_record_for_test_coins,
+        current_block_height=TEST_HEIGHT,
+        current_block_timestamp=TEST_TIMESTAMP,
+    )
+    conditions = [[ConditionOpcode.CREATE_COIN, IDENTITY_PUZZLE_HASH, 1], [opcode, lock_value]]
+    created_coin = Coin(TEST_COIN_ID, IDENTITY_PUZZLE_HASH, 1)
+    sb1 = spend_bundle_from_conditions(conditions)
+    sb2 = spend_bundle_from_conditions(conditions, created_coin)
+    # sb spends TEST_COIN and creates created_coin which gets spent too
+    sb = SpendBundle.aggregate([sb1, sb2])
+    # We shouldn't have a record of this ephemeral coin
+    assert await get_coin_record_for_test_coins(created_coin.name()) is None
+    # This ephemeral coin has entered the mempool at the current block height
+    assert mempool_manager.peak is not None
+    assert mempool_manager.peak.height == TEST_HEIGHT
+    _, status, error = await add_spendbundle(mempool_manager, sb, sb.name(), first_added_height=TEST_HEIGHT)
```

Comment:
I'm pretty sure `first_added_height` is irrelevant here. The fact that you specify it suggests that it matters. Does it? If so, I think it warrants a comment